src/test/java/org/broadinstitute/hellbender/utils/UtilsUnitTest.java

```diff
@@ -356,18 +356,17 @@ public void testUnsuccessfulValidIndex(final int index, final int length) {
     }
 
     @DataProvider(name = "unsuccessfulRegularReadableFileCheckData")
-    @SuppressWarnings("all")
     public Object[][] unsuccessfulRegularReadableFileCheckData() {
-        final File directory = createTempFile("Utils-RRFCD-Dir", ".dir");
-        directory.delete();
-        directory.mkdir();
+        final File directory = createTempDir("Utils-RRFCD-Dir");
         final File nonExistingFile = createTempFile("Utils-RRFCD-NoFile", ".file");
         nonExistingFile.delete();
         final File nonReadable = createTempFile("Utils-RRFCD-NoReadable", ".file");
-        nonReadable.setReadable(false);
-        return new Object[][]{
-                {directory}, {nonExistingFile}, {nonReadable}
-        };
+        final boolean successSetNotReadable = nonReadable.setReadable(false) && !nonReadable.canRead();
+        if (successSetNotReadable) {
```

Comment:
This would be nicer if it could generate a skip instead of fewer tests.  I don't see a clean way to do it though.  It's a bit weird for the total number of tests to change.  Maybe also add a log message when this happens?